docs/register-intune-device-id.md

```diff
@@ -19,25 +19,44 @@ Following are the prerequisites to register Intune device ID on TechPass portal:
 
```

Comment:
I couldn't add a comment so capturing it here. A few lines up there is a  service request link pointing to https://go.gov.sg/techpass-sr 
![image](https://github.com/GovTechSG/techpass-user-docs/assets/11743256/9731f61d-b410-4682-8580-aa055d7337dc)

Reply:
is that ok or...?